### DISCUSSION TITLE
The Bluetooth version...

### DISCUSSION THREAD

**Discussion by trip5** [EdenRidgway/XY-Wifi-Clock#9] [General]
I'm looking at the XY Clock with bluetooth here: https://www.aliexpress.com/item/1005002951814236.html

And quite frankly, it looks like a Blueooth module attached to an XY Clock.  The only points that meet between the two boards appear to be for power!

And I watched this video a bit too and it appears to be two different apps and I'm thinking the bluetooth receiver part might just have a "mode switch" or something in it's bluetooth command codes?  https://www.youtube.com/watch?v=PAnIIXp5Puo

Otherwise, the video or product page might have shown it's ability to play an MP3 from a USB stick as an alarm?  I mean, it still has a piezo speaker on the back!

And yeah I know it looks a bit different than the one here https://www.aliexpress.com/item/1005002933216217.html as on the Readme... but the solder points look the same... so I'm still guessing the lower board only supplies power to the clock board and they don't actually work in conjunction with each other...

**Comment by MacSass** [comment]
My guess is as well that the bluetooth part is fairly independent of the rest - it basically just routes the bluetooth audio stream to the aux connector. I would guess the clock part will still work with what you find here - although I have not tested it, as I have 2 modules without bluetooth. Why not give it a try to flash - just backup the original fw in case you are not happy with the replacement.
